data-serving/data-service/test/model/location.test.ts

```diff
@@ -18,7 +18,8 @@ describe('validate', () => {
         });
     });
 
-    it('a location without a geometry is invalid', async () => {
+    // Geometry will be added back in a new ticket
```

Comment:
OK to add all tests with `skip` back during #2974 ?

Reply:
Yes, I will add this to the description of #2974